docs/guide/01-Rendering-Halogen-HTML.md

```diff
@@ -197,3 +197,29 @@ html = HH.div [ HP.placeholder "blah" ] [ ]
 ```
 
 This error tells you that you've tried to use a property with an element that doesn't support it. It first lists the property you tried to use, and then it lists the properties that the element _does_ support. Another example of Halogen's type safety in action!
+
+### Adding missing properties
+
+Not all properties are currently available in Halogen.
```

Comment:
I think we should explain that HTML is a [living standard](https://html.spec.whatwg.org/multipage), which means that the standard continues to change over time and Halogen has to keep up with those changes. That means that Halogen and Web.HTML can end up a bit behind the standard sometimes and need to be updated to match it. That's the main reason why not all properties might be in the library.

Then we can clarify what you wrote: